OpenRA.Mods.Common/InstallUtils.cs

```diff
@@ -93,6 +68,10 @@ public static string GetMountedDisk(Func<string, bool> isValidDisk)
 						}
 					}
 
+					var dir = Path.GetDirectoryName(dest);
+					if (!Directory.Exists(dir))
+						Directory.CreateDirectory(dir);
```

Comment:
It looks like you can now remove the `Directory.CreateDirectory(destPath);` at the start of the method.